lib/mailcannon/reduces/envelope_bag_map.js

```diff
@@ -2,12 +2,8 @@ function () {
   values = {}
   values[this.event] = [this.target_id];
 
-  if (this.event == 'bounce') {
-  	if (this.type == 'bounce') {
-		emit(this.envelope_bag_id, { hard_bounce: [ this.target_id ]})
-	} else {
-		emit(this.envelope_bag_id, { soft_bounce: [ this.target_id ]})
-	}
+  if (this.event == 'bounce' && this.type == 'bounce') {
+	emit(this.envelope_bag_id, { hard_bounce: [ this.target_id ]})
```

Comment:
Identifier 'envelope_bag_id' is not in camel case.<br>Identifier 'hard_bounce' is not in camel case.<br>Identifier 'target_id' is not in camel case.<br>Missing semicolon.<br>'emit' is not defined.